build_debian.sh

```diff
@@ -352,6 +355,7 @@ sudo cp files/dhcp/rfc3442-classless-routes $FILESYSTEM_ROOT/etc/dhcp/dhclient-e
 sudo cp files/dhcp/sethostname $FILESYSTEM_ROOT/etc/dhcp/dhclient-exit-hooks.d/
 sudo cp files/dhcp/graphserviceurl $FILESYSTEM_ROOT/etc/dhcp/dhclient-exit-hooks.d/
 sudo cp files/dhcp/snmpcommunity $FILESYSTEM_ROOT/etc/dhcp/dhclient-exit-hooks.d/
+sudo cp files/dhcp/vrf $FILESYSTEM_ROOT/etc/dhcp/dhclient-exit-hooks.d/
```

Comment:
this has duplicated function with rfc3442-classless-routes, we should combine them.

Reply:
I think rfc3442-classless-routes can probably be removed.

Reply:
Understood. I am of the opinion that we should keep as is in order to have proper separation between vrf and non-vrf functionality (DNS functionality port to follow as well). It will also be easier to port any changes done at the source. Are you ok with that approach and leave as is?